packages/asset-service/src/generateAssetData/generateColorMap.ts

```diff
@@ -0,0 +1,53 @@
+import 'dotenv/config'
+
+import { AssetId } from '@shapeshiftoss/caip'
+import { Asset } from '@shapeshiftoss/types'
+import fs from 'fs'
```

Comment:
Might or might not tree-shake better
```suggestion
import fs from `fs/promises'
```

Reply:
Since it's a local script, I just kept as fs.